content/doc/pipeline/jenkinsfile.adoc

```diff
@@ -9,11 +9,11 @@ title: "Getting Started with Jenkinsfile"
 
 == Audience and Purpose
 
-This document is intended for novice Jenkins users who want to leverage the power of pipeline functionality. Extending the reach of what was learned from a "Hello World" example in link:http://jenkins.io/doc/pipeline/[Getting Started with Pipelines], this document explains how to use a Jenkinsfile to perform a simple checkout and build for the contents of a repository. 
+This document is intended for novice Jenkins users who want to leverage the power of pipeline functionality. Extending the reach of what was learned from a "Hello World" example in link:http://jenkins.io/doc/pipeline/[Getting Started with Pipelines], this document explains how to use a Jenkinsfile to perform a simple checkout and build for the contents of a repository.
```

Comment:
This isn't for you @cyrille-leclerc, but @PaddyOH I appeared to have missed this in the first pass of review. Relative URLs are preferred over domain-specific ones. E.g. `/doc/pipeline` instead of including the full jenkins.io